libraries/wallet/wallet.cpp

```diff
@@ -234,6 +234,8 @@ fc::optional<fc::variant> wallet_api::get_htlc(std::string htlc_id) const
          { return convert("SHA1", obj.str()); }
          result_type operator()(const fc::sha256& obj)const
          { return convert("SHA256", obj.str()); }
+         result_type operator()(const fc::hash160& obj)const
+         { return convert("HASH160", obj.str()); }
```

Comment:
Since this is the only modified cpp file in wallet, I'm putting the issues here.
* Need to update the `htlc_create` wallet command or add a new command to include the new `memo` field. Ideally we need to consider backward compatibility when updating commands to not break existing client applications, however I guess this time it's not a big issue (are there any serious client apps using this command?).
* Need to extend the `do_hash` function to support HASH-160.